game/src/options.rs

```diff
@@ -6,6 +6,9 @@ use ezgui::{
 };
 use geom::Duration;
 
+const PANNING_MAP_DISABLE_STR: &'static str =
```

Comment:
I don't find creating an extra constant for this sort of thing to be more readable / maintainable when there are just two uses, both in this file. It is a bit much to repeat in `is_checked`. An alternative is to add `.named("disable pan")` to `Checkbox::text` and then inline that simpler string twice. You'll have to delete the assertion at the beginning of `named` in `ezgui/src/managed.rs` if so -- I think it's reasonable to allow widget renaming.

(The more general code style guideline is to be consistent with existing stuff. If you spot a pattern that's not great, we should fix it everywhere)